src/i18n/locales/ca.json

```diff
@@ -670,16 +678,16 @@
       "edit_my_organization": "La meva entitat",
       "votings": "Votacions"
     },
-    "overview": "Overview",
-    "required_signin": "You must sign in first",
+    "my_entity": "La meva entitat",
+    "overview": "Visió general",
+    "required_signin": "Primer has d'iniciar sessió",
     "title": {
-      "edit": "Editar organització",
       "read_more": "Expandir títol"
     },
-    "view_public_profile": "View public profile",
-    "voting_processes": "{{votes}} voting processes",
-    "votings": "Votings",
-    "votings_overview": "Votings overview"
+    "voting_processes_one": "{{ count }} procés de votació",
+    "voting_processes_many": "{{ count }} processos de votació",
+    "voting_processes_other": "{{ count }} processos de votació",
```

Comment:
One of these two is redundant and unnecessary (not sure which one, but you can check by just removing one and then the other).